discovery-provider/plugins/network-monitoring/src/metrics/queries.ts

```diff
@@ -213,3 +213,38 @@ export const getUsersWithNullPrimaryClock = async (run_id: number): Promise<numb
 
     return usersCount
 }
+
+export const getUsersWithAllFoundationNodeReplicaSetCount = async (run_id: number): Promise<number> => {
```

Comment:
can just hardcode spIDs like we talked about

Reply:
I've playing around with this a bit more and I don't think hard coding the SpIDs is going to work because SpIDs are different in staging and prod. Although the current query is kinda a mess, I think it's more robust because it only relies on the network monitoring DB.

Reply:
but we have stage and prod envs so we can hardcode the spIDs as an envvar no? would be much simpler

Reply:
alternatively we don't have to export a metric here, but can build a chart in grafana directly where we hardcode the spIDs there, what do you think?

Reply:
I looked into it a bit more and we unfortunately don't have a metric in prometheus that would let use filter like that. We have to create a new one. 


> but we have stage and prod envs so we can hardcode the spIDs as an envvar no? would be much simpler

We could do it like this but idk if giving metrics the knowledge of what ENV they're running in is the best way to do it. It seems stabler to have metrics get information from the network monitoring DB